src/templates.rs

```diff
@@ -1,18 +1,17 @@
-extern crate pencil;
-extern crate motivations;
-extern crate pick_one;
-
-use std::collections::BTreeMap;
-
+use handlebars::Handlebars;
 use motivations::MOTIVATIONS;
+use pick_one;
+use std::collections::BTreeMap;
 
-use pencil::{Request, PencilResult};
-
-pub fn motivation(request: &mut Request) -> PencilResult {
+pub fn motivation() -> Vec<u8> {
```

Comment:
return a result?

Reply:
I thought this would be easy, but it isn't, until we port simple-server to error-chain/failure

basically, handlebars' errors don't implement our error trait, and we don't have a good error for it to map to